www/__tests__/metricsHelper.test.ts

```diff
@@ -1,51 +1,56 @@
+import { DateTime } from 'luxon';
 import {
   calculatePercentChange,
   formatDate,
   formatDateRangeOfDays,
   getLabelsForDay,
   getUniqueLabelsForDays,
+  secondsToHours,
+  secondsToMinutes,
   segmentDaysByWeeks,
+  metricToValue,
+  tsForDayOfMetricData,
+  valueForFieldOnDay,
+  generateSummaryFromData,
+  isCustomLabels,
+  isAllCustom,
+  isOnFoot,
+  getUnitUtilsForMetric,
 } from '../js/metrics/metricsHelper';
-import {
-  DayOfClientMetricData,
-  DayOfMetricData,
-  DayOfServerMetricData,
-} from '../js/metrics/metricsTypes';
+import { DayOfMetricData } from '../js/metrics/metricsTypes';
+import initializedI18next from '../js/i18nextInit';
+window['i18next'] = initializedI18next;
 
 describe('metricsHelper', () => {
   describe('getUniqueLabelsForDays', () => {
     const days1 = [
-      { label_a: 1, label_b: 2 },
-      { label_c: 1, label_d: 3 },
-    ] as any as DayOfServerMetricData[];
-    it("should return unique labels for days with 'label_*'", () => {
+      { mode_confirm_a: 1, mode_confirm_b: 2 },
+      { mode_confirm_b: 1, mode_confirm_c: 3 },
+      { mode_confirm_c: 1, mode_confirm_d: 3 },
+    ] as any as DayOfMetricData[];
+    it("should return unique labels for days with 'mode_confirm_*'", () => {
       expect(getUniqueLabelsForDays(days1)).toEqual(['a', 'b', 'c', 'd']);
     });
-
-    const days2 = [
-      { mode_a: 1, mode_b: 2 },
-      { mode_c: 1, mode_d: 3 },
-    ] as any as DayOfClientMetricData[];
-    it("should return unique labels for days with 'mode_*'", () => {
-      expect(getUniqueLabelsForDays(days2)).toEqual(['a', 'b', 'c', 'd']);
-    });
```

Comment:
@jiji14 please replace this with a test that actually have duplicate labels (see my comment in https://github.com/e-mission/e-mission-phone/pull/1138#discussion_r1607576184)